data/yam/agama/auto/lib/storage.libsonnet

```diff
@@ -182,7 +182,52 @@ local raid(level='raid0', uefi=false) = {
     },
   ],
 };
+
+local iscsi_activation_non_root() = {
```

Comment:
actually 'activation' should be a word in this variable name (sorry for my previous advice). We could go with:
`home_on_iscsi`